corehq/ex-submodules/couchexport/writers.py

```diff
@@ -554,3 +554,50 @@ class ZippedHtmlExportWriter(ZippedExportWriter):
     writer_class = HtmlFileWriter
     table_file_extension = ".html"
     format = Format.ZIPPED_HTML
+
+
+class GeoJSONWriter(JsonExportWriter):
+    format = Format.GEOJSON
+    table_file_extension = ".geojson"
+
+    @staticmethod
+    def parse_feature(coordinates, properties):
+        return {
+            "type": "Feature",
+            "geometry": {"type": "Point", "coordinates": coordinates},
+            "properties": properties,
+        }
+
+    def get_features(self, table, data):
+        geo_property_name = table.selected_geo_property
+        table_headers = data[0]
+        if geo_property_name not in table_headers:
+            return []
+
+        geo_data_index = table_headers.index(geo_property_name)
+        features = []
+        for row in data[1:]:
+            try:
+                lat, lng, _, _ = row[geo_data_index].split(" ")
+            except ValueError:
+                continue
+            properties = {header: row[i] for i, header in enumerate(table_headers) if header != geo_property_name}
+            features.append(self.parse_feature(
+                coordinates=[lng, lat],
+                properties=properties,
+            ))
+        return features
+
+    def _close(self):
+        feature_collections = []
+        for table, data in self.tables.items():
```

Comment:
We loop through each table in the export, however a few lines below we only grab the first table's features. Do we require this loop if we're only interested in the first table?